docs/ide/dotfuscator/install.md

```diff
@@ -16,62 +17,65 @@ helpviewer_keywords:
   - "install Dotfuscator"
   - "installing Dotfuscator"
   - "set up Dotfuscator"
-description: "Learn how to install the free Dotfuscator Community Edition included in Visual Studio 2017."
+description: "Learn how to install the free copy of Dotfuscator Community included in Visual Studio 2019."
 ms.assetid: f2146651-e24a-4e24-ade8-8ddee8ff4e43
 author: "Joe-Sewell-PreEmptive"
 ms.author: gewarren
 manager: jillfra
 ---
-# Install Dotfuscator Community Edition (CE)
+# Install Dotfuscator Community
 
-Visual Studio 2017 introduces a new low-impact installation experience.
-As a result, Dotfuscator Community Edition (Dotfuscator CE) is not installed by default.
-However, it is easy to install Dotfuscator CE even if you have already installed Visual Studio.
+Visual Studio editions 2017 and newer no longer come with Dotfuscator Community installed by default.
+However, installing Dotfuscator Community on your copy of Visual Studio is a simple process.
 
 > [!NOTE]
-> In addition to the versions of Dotfuscator CE shipped with releases of Visual Studio,
+> In addition to the versions of Dotfuscator Community shipped with releases of Visual Studio,
 > PreEmptive Solutions also periodically provides updated versions on its website.
 > If you want to download the **latest version** directly instead of installing from Visual Studio,
 > **[click here to go to the Dotfuscator Downloads page][download]**.
 
-## Within Visual Studio
 
-You can install Dotfuscator CE from the Visual Studio IDE:
+## Within Visual Studio 2019
```

Comment:
We are sharing files for both VS 2017 and VS 2017. There are ways to make an entire file applicable to just one or the other version, or to just make certain sections of a file applicable to one version. Would it be better to make this version agnostic if there's no difference between VS 2017 and VS 2019? If you want the syntax for making files or zones of files specific to a version, let me know.

Reply:
I think it would probably be best to make version-specific zones for the installation section, could you give me an example of the syntax for separating zones of files? Thanks!

Reply:
Take a look at [this file](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/get-started/includes/ide-overview.md) for an example. Look for the sections surrounded by `::: moniker range=""` and `::: moniker-end`.

Reply:
Great, thanks!